doctor/crud.go

```diff
@@ -20,20 +20,19 @@ func CreateDoctor(w http.ResponseWriter, r *http.Request) {
 	decoder := json.NewDecoder(r.Body)
 	decoder.Decode(&doc)
```

Comment:
don't ignore error here and return suitable status code

Reply:
done